drivers/dma/dmamux_stm32.c

```diff
@@ -45,6 +52,36 @@ struct dmamux_stm32_config {
 	const struct dmamux_stm32_channel *mux_channels;
 };
 
+/*
+ * UTIL_LISTY is used to generte arrays with function pointers to check
+ * and clear interrupt flags using LL functions
+ */
+#define DMAMUX_CHANNEL(i, _)		LL_DMAMUX_CHANNEL_ ## i,
+#define IS_ACTIVE_FLAG_SOX(i, _)	LL_DMAMUX_IsActiveFlag_SO  ## i,
+#define CLEAR_FLAG_SOX(i, _)		LL_DMAMUX_ClearFlag_SO ## i,
+#define IS_ACTIVE_FLAG_RGOX(i, _)	LL_DMAMUX_IsActiveFlag_RGO  ## i,
+#define CLEAR_FLAG_RGOX(i, _)		LL_DMAMUX_ClearFlag_RGO ## i,
+
+uint32_t table_ll_channel[] = {
+	UTIL_LISTIFY(DT_INST_PROP(0, dma_channels), DMAMUX_CHANNEL)
+};
+
+uint32_t (*func_ll_is_active_so[])(DMAMUX_Channel_TypeDef * DMAMUXx) = {
```

Comment:
As raised in another pr, the pointer asterisk should be next to the name.
But checkpatch seems to interpret this as multiplication(ctx:WxV)  so I had to change it here to comply with the check.
Grepping for `\*\ const` shows about 500 occurrences, so am I right in assuming that this not so strict? How should such false positives be handled?

Reply:
For false positive, you should raise the point to merging people, so the PR is merged despite failing CI.